cmd/erasure-createfile_test.go

```diff
@@ -54,7 +54,6 @@ var erasureCreateFileTests = []struct {
 	{dataBlocks: 7, onDisks: 14, offDisks: 7, blocksize: int64(blockSizeV1), data: oneMiByte, offset: 0, algorithm: DefaultBitrotAlgorithm, shouldFail: false, shouldFailQuorum: true},                 // 9
 	{dataBlocks: 8, onDisks: 16, offDisks: 8, blocksize: int64(blockSizeV1), data: oneMiByte, offset: 0, algorithm: DefaultBitrotAlgorithm, shouldFail: false, shouldFailQuorum: true},                 // 10
 	{dataBlocks: 5, onDisks: 10, offDisks: 3, blocksize: int64(oneMiByte), data: oneMiByte, offset: 0, algorithm: DefaultBitrotAlgorithm, shouldFail: false, shouldFailQuorum: false},                  // 11
-	{dataBlocks: 6, onDisks: 12, offDisks: 5, blocksize: int64(blockSizeV1), data: oneMiByte, offset: 102, algorithm: 0, shouldFail: true, shouldFailQuorum: false},                                    // 12
```

Comment:
It is not readily clear why this test case is being deleted (also what about the test case numbering comment that will no longer be valid?) - and there are multiple such test case deletions in this PR. Could you note the reason for the deletions in the PR?

Reply:
Previously storage.CreateFile() took algo type as an argument and we had -ve test for invalid algo type.

Now storage.CreateFile() does not take algo type as an argument, but instead takes `bitrotWriter` and `bitrotReader`.

newBitrotReader() and newBitrotWriter() does not have -ve test cases for invalid algo type because if it is passed with invalid algo type, it os.Exits in logger.CriticalIf() :
```
59 func (a BitrotAlgorithm) New() hash.Hash {                                                                                                                                                                                                                                  
60         switch a {                                                                                                                                                                                                                                                          
61         case SHA256:                                                                                                                                                                                                                                                        
62                 return sha256.New()                                                                                                                                                                                                                                         
63         case BLAKE2b512:                                                                                                                                                                                                                                                    
64                 b2, _ := blake2b.New512(nil) // New512 never returns an error if the key is nil                                                                                                                                                                             
65                 return b2                                                                                                                                                                                                                                                   
66         case HighwayHash256:                                                                                                                                                                                                                                                
67                 hh, _ := highwayhash.New(magicHighwayHash256Key) // New will never return error since key is 256 bit                                                                                                                                                        
68                 return hh                                                                                                                                                                                                                                                   
69         default:                                                                                                                                                                                                                                                            
70                 logger.CriticalIf(context.Background(), errors.New("Unsupported bitrot algorithm"))                                                                                                                                                                         
71                 return nil                                                                                                                                                                                                                                                  
72         }                                                                                                                                                                                                                                                                   
73 }                                                                                                                                                                                                                                                                           
```

But we will never have to deal with this situation because the only place an invalid algo type input can happen if there is an invalid algo in xl.json - in which case we fail during the unmarshall itself.

I have fixed the test numbering order.